.travis.yml

```diff
@@ -1,5 +1,10 @@
+sudo: false
+
 language: node_js
 
+node_js:
+  - "node"
```

Comment:
Ok, so a version has to be given here. And using "node" will pick the latest release. I think that this is fine.